### ISSUE TITLE
Getting IREEs pack based flow working to handle the OPT dispatches: Reductions

### ISSUE THREAD

**Issue by powderluv** [nod-ai/iree-amd-aie#152]

**Comment by MaheshRavishankar** [comment]
Separate issue to track status of compiling OPTs reduction op based dispatches (original collection of all dispatches https://github.com/nod-ai/iree-amd-aie/issues/96 ).

End state here should be that these dispatches should run end-to-end and tested on CI.

Blocked on https://github.com/nod-ai/iree-amd-aie/issues/146 and on general codegeneration support built in amd-aie backend.
